### ISSUE TITLE
Bug after clicking "Fork This Skeleton Here"

### ISSUE THREAD

**Issue by trejder** [jsfiddle/jsfiddle-issues#614]
You clearly have a bug around `Fork This Skeleton Here` link. See below -- an effect of clicking that link... several times:

![fork this skeleton here](https://cloud.githubusercontent.com/assets/2903392/8327310/aba15e78-1a67-11e5-805f-6f2b13576920.png)

**Comment by baconface** [comment]
![Yo dawg](https://i.imgflip.com/nc05g.jpg)

**Comment by oskarkrawczyk** [comment]
@trejder How did you manage to get it like this?

**Comment by trejder** [comment]
@oskarkrawczyk Sorry for not putting this straight. It is under `Fork This Skeleton Here` link. It does not have `target=_new` in HTML code (as I assume, didn't check this) and opens target in the same window.

Anyway, this is how I did manage to get this -- I simply clicked on `Fork This Skeleton Here` link few times, each time in a new "copy" of JSFiddle. This is two months old issue so I don't remember details.
